### PR TITLE
TW-142 Notification page manager side

### PR DESCRIPTION
**Pull request by lendihop**
https://madfish.atlassian.net/jira/software/projects/TW/boards/4?selectedIssue=TW-142

### CODE REVIEW

File: db.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
+{
```
[the remark this answers is not in the document]
Comment (lendihop) [reply db.json:1]: yes. Thats why we need to add it to gitignore after the first deploy

File: redis.conf
Comment (lendihop) [inline redis.conf:1]: disables redis RDB and AOF backups, https://redis.io/docs/management/persistence/

File: src/notifications/utils/get-notifications.util.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { LowdbSync } from 'lowdb';
+
+import { DbData } from '../../interfaces/db-data.interface';
+import { MANDATORY_NOTIFICATIONS_LIST } from '../mandatory-notifications-list.data';
+import { PlatformType } from '../notification.interface';
+import { clearExpiredNotifications } from './clear-expired-notifications.util';
+
+export const getNotifications = (db: LowdbSync<DbData>, platform: PlatformType, startFromTime: number) => {
```
[the remark this answers is not in the document]
Comment (lendihop) [reply src/notifications/utils/get-notifications.util.ts]: the problem is that startFromTime parameter is different for every user, so i suppose we wouldnt get much from such memoization + cache size of this fn would be really big

**Review by lourenc — CHANGES_REQUESTED** [review]

File: src/middlewares/basic-auth.middleware.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import { Request, Response, NextFunction } from 'express';
+
+import { ADD_NOTIFICATION_PASSWORD, ADD_NOTIFICATION_USERNAME } from '../config';
+import { isDefined } from '../utils/helpers';
+
+export const basicAuth = (req: Request, res: Response, next: NextFunction) => {
+  if (isDefined(req.get('Authorization'))) {
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```
Comment (lourenc) [inline src/middlewares/basic-auth.middleware.ts:8]: Can we remove this eslint disable rule? 

In order to do so, you can assign `req.get('Authorization')` to a variable and then use `isDefined` as a typeguard on the variable value.

**Review by lourenc — CHANGES_REQUESTED** [review]

**Review by lourenc — APPROVED** [review]

**Pull request merged**